App/authPopup.js

```diff
@@ -75,13 +75,14 @@ function signOut() {
      * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/request-response-object.md#request
      */
 
-    // Choose which account to logout from.
-    
     const logoutRequest = {
-        account: myMSALObj.getAccountByHomeId(accountId)
+        postLogoutRedirectUri: msalConfig.auth.redirectUri,
+        redirectMainWindowTo: msalConfig.auth.redirectUri
     };
     
-    myMSALObj.logout(logoutRequest);
+    myMSALObj.logoutPopup(logoutRequest).then(() => {
+        window.location.reload();
```

Comment:
This may not be needed if you're redirecting the main window with the request

Reply:
reloading the page? that's what I expected but the main window stayed the same after logout with popup..

Reply:
my bad, it wasn't working 'cause I had the property name wrong in `logoutRequest`. removing..